### PR TITLE
CWP DOMA: import fixes from Overleaf

### PR DESCRIPTION
**Pull request by jouvin**
- Changes made up to 13/4 included

### CODE REVIEW

File: CWP/papers/HSF-CWP-2017-04_doma/latex/hsf-cwp-dm.tex [hunk 1]
```diff
@@ -41,49 +41,35 @@
 
 \section{Introduction}\label{introduction}
 
-The coming decade will bring HEP to the exabyte (EB) scale with the expected data
+The coming decade will bring HEP to the exabyte scale with the expected data
 volumes of the HL-LHC experiments, the DUNE experiment, and even non-HEP
-experiments such as the Square Kilometre Array (SKA) entering this
-regime. In devising experimental computing models for this era, many
+experiments such as the Square Kilometer Array (SKA) entering this
```
Comment (graeme-a-stewart) [inline CWP/papers/HSF-CWP-2017-04_doma/latex/hsf-cwp-dm.tex]: Again, wrong - it's "Square Kilometre Array"!

File: CWP/papers/HSF-CWP-2017-04_doma/latex/hsf-cwp-dm.tex [hunk 2]
```diff
@@ -156,77 +133,61 @@ \section{Current Approaches}\label{current-approaches}
 
 \begin{itemize}
 \item \textbf{Data Organization:} This is essentially how data is structured
-  as it is written. Most data is written in flat files, in the ROOT format, and
-  typically with a column-wise organization of the data. The records
-  corresponding to these columns are compressed. The internal details of
-  this organization are typically visible only to individual software
-  applications.
+  as it is written. Most data is written in the ROOT file format. ROOT serializes data objects, compresses, and writes them in a column-wise manner. The data written in this way can only be meaningfully read  by a software that includes data object libraries.
```
Comment (graeme-a-stewart) [inline CWP/papers/HSF-CWP-2017-04_doma/latex/hsf-cwp-dm.tex]: small correction: "meaningfully read by ~~a~~ software"

File: CWP/papers/HSF-CWP-2017-04_doma/latex/hsf-cwp-dm.tex [hunk 3]
```diff
@@ -257,9 +217,9 @@ \subsection{HEP Workflow in Context}\label{hep-workflow-in-context}
   resources) and exploiting the available data bandwidth between the
   storage and the compute locations. Data replication also serves to
   guard against data loss. Data replication places modest requirements
-  on the rate of IOP transactions that need to be supported, but drives
+  on the IOPS that need to be supported, but drives
   the bandwidth requirements that are needed in enacting site-to-site
-  data transfers between WAN endpoints (e.g. 10-100~Gbps). This
+  data transfers between WAN endpoints (\emph{e.g.}, 10-100~Gbps). This
```
Comment (graeme-a-stewart) [inline CWP/papers/HSF-CWP-2017-04_doma/latex/hsf-cwp-dm.tex]: You need to be consistent in italicising (or not) Latinate abbreviations. (I would suggest not, as per most modern style guides and observing that in most other places in the document you don't.)

File: CWP/papers/HSF-CWP-2017-04_doma/latex/hsf-cwp-dm.tex [hunk 4]
```diff
@@ -509,7 +461,7 @@ \section{New Analysis Paradigms}\label{new-analysis-paradigms}
 products. We also need to take into account that the adoption of these
 techniques, if they prove to be beneficial, would represent a disruptive
 change which directly impacts the end user and therefore promoting
-acceptance through intermediate solutions would be desirable.
+acceptance through intermediate solutions would be desirable. One such solution - ``Service X'' - has been proposed in \href{https://bit.ly/2Ix9NIL}{Organizing Data Lakes}. The service would start with the data in its persistent format, perform one or more operations on it  (\emph{e.g.} joins, query, decompression, filtering to sub-event level), and finally deliver a data stream to a client.
```
Comment (graeme-a-stewart) [inline CWP/papers/HSF-CWP-2017-04_doma/latex/hsf-cwp-dm.tex]: Be consistent in italicising, as above.

File: CWP/papers/HSF-CWP-2017-04_doma/latex/hsf-cwp-dm.tex [hunk 5]
```diff
@@ -603,10 +555,22 @@ \section{Common Challenges}\label{common-challenges}
 which has not yet been adopted widely by the field.
 
 HEP experiments should be prepared to leverage ``tactical storage''.
-
-Storage that becomes most cost-effective as it becomes available (e.g.,
+Such storage may be provisioned only when it becomes cost-effective
+(e.g.,
+% * <briedel@uchicago.edu> 2018-04-05T15:43:01.526Z:
```
Comment (graeme-a-stewart) [inline CWP/papers/HSF-CWP-2017-04_doma/latex/hsf-cwp-dm.tex]: Do you really want to keep discussion comments now?

File: CWP/papers/HSF-CWP-2017-04_doma/latex/hsf-cwp-dm.tex [hunk 6]
```diff
@@ -619,11 +583,17 @@ \section{Common Challenges}\label{common-challenges}
...
+% > stability.
+% This paragraph begs the question of whether one needs to move to "direct-access-based archives" if the data is only accessed once or twice. The plenty of tape is not that great in that case.
+%
+% ^.
 
 Cost reductions in maintenance and operation of the storage
 infrastructure can be realized through convergence of the major
 experiments and resource providers on shared solutions. This does not
-necessarily mean promoting a monoculture, as different solutions will be
+necessarily mean promoting a mono-culture, as different solutions will be
```
Comment (graeme-a-stewart) [inline CWP/papers/HSF-CWP-2017-04_doma/latex/hsf-cwp-dm.tex]: Revert this - the single word "monoculture" is correct.

File: CWP/papers/HSF-CWP-2017-04_doma/latex/hsf-cwp-dm.tex [hunk 7]
```diff
@@ -681,10 +651,10 @@ \subsection{Data organization and analysis
   decompression schemes and mappings onto hardware architectures
   considering the flow of data from spinning disk to memory and
   application.
-\item Evaluate possible alternative storage formats to the ROOT format, esp.\ storage formats optimized for storage efficiency and/or  data filterting through metdata
+\item Evaluate possible alternative storage formats to the ROOT format, esp.\ storage formats optimized for storage efficiency and/or  data filtering through metadata
```
Comment (graeme-a-stewart) [inline CWP/papers/HSF-CWP-2017-04_doma/latex/hsf-cwp-dm.tex]: Curious  - why write "esp." and not just "especially".

File: CWP/papers/HSF-CWP-2017-04_doma/latex/hsf-cwp-dm.tex [hunk 8]
```diff
@@ -733,18 +702,16 @@ \subsection{Global optimization of efficiency and latency}\label{efficiency-late
...
 
 \subsection{Data Access and Data Access Patterns}\label{access-patterns}
-The data access pattern should inform the the use of storage media and data location. Proposed areas of inquiry are:
+The data access needs and patterns should inform the the choice of storage media and data location. Proposed areas of inquiry are:
 
 \begin{itemize}
 \item Understand data access patterns, including investigating frequency of access of data
 \item Exploring ways to ease preliminary data filtering steps
-\item \end{itemize}
+\item Exploring commercially-popular data transfer methods (\emph{e.g,} HTTP)
```
Comment (graeme-a-stewart) [inline CWP/papers/HSF-CWP-2017-04_doma/latex/hsf-cwp-dm.tex]: Questionable italicisation again.

**Comment by jouvin** [comment]
@graeme-a-stewart thanks for your remark, I'll implement them. About SKA, the text is in American English and the main author probably missed that SKA was not an American acronym!

**Comment by jouvin** [comment]
@graeme-a-stewart at this point, probably easier to wait the submission to arXiv... I'll fix the conflict with the layout change.

**Comment by graeme-a-stewart** [comment]
Ping on this

**Comment by jouvin** [comment]
No news from Bo... We can probably merge this one anyway and will open a new one for the final fixes....

**Comment by graeme-a-stewart** [comment]
OK, let's do that then...

**Pull request merged**